getting-started/proc-add-custom-app-file-openshift.adoc

```diff
@@ -30,15 +30,34 @@ data:
 . Go to the *Helm* tab.
 +
 The list of Helm Releases appears on the page.
-. Click the three dots of a Helm release and select *Upgrade*.
-. Add the following values in *Root Schema -> Backstage Chart Schema ->  Backstage Parameters -> Extra App Configuration files to inline into command arguments*:
+. Click the three dots corresponding to a Helm release and select *Upgrade*.
```

Comment:
IBM style guide says to refer to this icon by its tooltip or as an _overflow menu_.